packages/hardhat-core/sample-project/test/sample-test.js

```diff
@@ -4,11 +4,13 @@ describe("Greeter", function() {
   it("Should return the new greeting once it's changed", async function() {
     const Greeter = await ethers.getContractFactory("Greeter");
     const greeter = await Greeter.deploy("Hello, world!");
-    
     await greeter.deployed();
+
     expect(await greeter.greet()).to.equal("Hello, world!");
 
-    await greeter.setGreeting("Hola, mundo!");
+    const setGreetingTx = await greeter.setGreeting("Hola, mundo!");
+    await setGreetingTx.wait();
```

Comment:
Maybe dd a comment here? Most people don't do this.

Reply:
Makes sense, added.